src/DCRoom.php

```diff
@@ -329,6 +329,42 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    public static function rawSearchOptionsToAdd()
+    {
+        $tab = [];
+
+        // separator
+        $tab[] = [
+            'id'   => 'dcroom',
+            'name' => self::getTypeName(1),
+        ];
+
+        $tab[] = [
+            'id'                 => '1500',
```

Comment:
A bit huge maybe?

Reply:
with ```php tools/getsearchoptions.php --type=Computer``` the last ```id``` is 1432 (For ```Phone```)

![image](https://github.com/glpi-project/glpi/assets/7335054/c98309ae-1cf3-45f6-be9a-5aeeb61c53c3)

the last commit lowers the ID to 1450